library/core/src/main/java/com/google/android/exoplayer2/ExoPlayerImplInternal.java

```diff
@@ -1041,8 +1068,9 @@ private void doSomeWork() throws ExoPlaybackException, IOException {
   }
 
   private long getCurrentLiveOffsetUs() {
-    return getLiveOffsetUs(
-        playbackInfo.timeline, playbackInfo.periodId.periodUid, playbackInfo.positionUs);
+    return presentationLatencyMs == 0 ? getLiveOffsetUs(
```

Comment:
should we just let it return the real live offset?

Reply:
`presentationLatencyMs` is actually encoder latency calculated from PRFT. `getLiveOffsetUs` is something else

Reply:
I guess it is currently set by wraper player.
Probably we should move it to set by Exoplayer internally